doc/source/index.rst

```diff
@@ -30,7 +30,7 @@ ICESat-2 datasets to enable scientific discovery.
     ---
     :img-top: https://cdn-icons-png.flaticon.com/128/2498/2498074.png
 
-    **Getting Started**
+    **Getting started**
```

Comment:
Thoughts on capitalization? It looks like we've got multiple conventions going (I don't have strong feelings about one or the other).